src/jit/rationalize.cpp

```diff
@@ -605,6 +605,9 @@ void Rationalizer::RewriteAssignment(LIR::Use& use)
             }
             JITDUMP("Rewriting GT_ASG(%s(X), Y) to %s(X,Y):\n", GenTree::NodeName(location->gtOper),
                     GenTree::NodeName(storeOper));
+#if NODEBASH_STATS
+            GenTree::RecordOperBashing(storeBlk->OperGet(), storeOper);
+#endif
             storeBlk->gtOper = storeOper;
             storeBlk->gtFlags &= ~GTF_DONT_CSE;
```

Comment:
I have no idea - I didn't want to mess with any existing code whenever possible, I could see no reason why SetOper wasn't being used, but then SetOper has been around for over 20 years so someone presumably had to have a reason not to call it?